dashboard/src/app/(authenticated)/(diagram)/components/NodeTypes/UserTask/UserTask.tsx

```diff
@@ -26,11 +29,10 @@ const Node: FC<NodeProps> = ({ data, selected }) => {
             </Link>
           </div>
           <div className="mb-2 flex gap-2 text-nowrap">
-            {userTask.userGroup && (
-              <div className="flex items-center justify-center">Group: {getVariable(userTask.userGroup)}</div>
-            )}
-            {userTask.userId && (
-              <div className="flex items-center justify-center">User: {getVariable(userTask.userId)}</div>
+            {nodeRun ? (
+              <UserTaskRunDetails userTaskNode={userTask} nodeRun={nodeRun} />
```

Comment:
This component should handle `nodeRun` with `undefined` value. Please handle the case on a parent component to simplify this one.

Reply:
Mijail are you saying that I should have a component that receives the nodeRun and handle both cases internally instead of handling them at the UserTask.tsx?